runtime/imports.go

```diff
@@ -652,16 +653,16 @@ func ext_sr25519_verify(context unsafe.Pointer, msgData, msgLen, sigData, pubkey
 	msg := memory[msgData : msgData+msgLen]
 	sig := memory[sigData : sigData+64]
 
-	pub, err := crypto.NewSr25519PublicKey(memory[pubkeyData : pubkeyData+32])
+	pub, err := sr25519.NewPublicKey(memory[pubkeyData : pubkeyData+32])
 	if err != nil {
 		return 1
 	}
 
-	if pub.Verify(msg, sig) {
-		return 0
+	if ok, err := pub.Verify(msg, sig); err != nil || !ok {
```

Comment:
I wonder if we can make this cleaner. Maybe update `Verify` to only return ok if no error.

Reply:
I could change it to this:
```
	ok, err := pub.Verify(msg, sig)
	if err != nil || !ok {
		return 1
	}
```

Verify has to return 1 if the call fails for any reason, 0 otherwise as per spec, so not sure how much I can change this